wordpress-mobile/WordPress-iOS
# Accessibility: Adjust collapsable header text sizes

**Fixes** https://github.com/wordpress-mobile/WordPress-iOS/issues/15925

## To test:
1. Set your device to a larger text size (The screenshots max out the size by enabling "Larger text sizes" in Accessibility Settings)
2. Navigate to Choose a Design and Choose A Domain
3. Notice that the header now accepts taps to scroll the page. The search box is above the keyboard

| Before | After |
| --- | --- |
| <img width="300" src="https://user-images.githubusercontent.com/3384451/126844189-2ca32bc3-3724-4257-bcf5-f4715d4db2e6.png"> | <img width="300" src="https://user-images.githubusercontent.com/3384451/126844222-b47411fd-b4f9-42f7-b8be-a6ed82610cca.png"> |
| <img width="300" src="https://user-images.githubusercontent.com/3384451/126844242-958f467a-b8d8-48a9-8193-ab7e65457d81.png"> | <img width="300" src="https://user-images.githubusercontent.com/3384451/126844283-69801c6c-098a-4a56-b634-8d82d3d38583.png"> |

## Regression Notes
1. Potential unintended areas of impact
None

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

3. What automated tests I added (or what prevented me from doing so)
None

## PR submission checklist:
